torch/quantization/convert_modules.py

```diff
@@ -0,0 +1,221 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch.nn as nn
+import torch.nn.quantized as nnq
+import torch
+from .observer import *
+
+def _forward_hook(self, input, output):
+    for module in self.children():
```

Comment:
wait, why is this the forward hook? why it can't just explicitly call .observer ?

Reply:
this is for modifying the behavior of forward function, isn't that needed?

Reply:
meaning why are we iterating over .children()? can't you just do `self.observer(output)`?